src/urbit/app/chess.hoon

```diff
@@ -653,9 +654,15 @@
         ?~  archived-game  ~
         ``[%chess-game !>(u.archived-game)]
       ``[%chess-game !>(game.u.active-game)]
-    ::
-    ::  .^(arch %gy /=chess=/game)
-    ::  collect all the game-id keys
+  ::
+  ::  .^(noun %gx /=chess=/mutuals/noun)
+  ::  .^(json %gx /=chess=/mutuals/json)
+  ::  read mutual pals
+    [%x %mutuals ~]
+      ``[%chess-pals !>((~(mutuals pals bowl) ~.))]
+  ::
+  ::  .^(arch %gy /=chess=/game)
+  ::  collect all the game-id keys
```

Comment:
This wasn't added in this commit, just moved over to the left. I've amended the indentation of the comments here to match the style we see in `/sur/chess`'s definition of `chess-game` and `chess-position`. If we didn't have the comments here, I'd keep the single `::` runes separating the scry paths, which is why I've indented the comments this way.

Reply:
What do ya know, you're right: this is the correct commenting style for kingside tall format (which we're also doing incorrectly). Okay, fair enough. Feel free to correct the style on any code you touch, and we'll do a style refactor after milestone 1, when we have a set style guide.